storm-client/test/jvm/org/apache/storm/grouping/LoadAwareShuffleGroupingTest.java

```diff
@@ -385,12 +371,12 @@ private int[] runChooseTasksWithVerification(LoadAwareShuffleGrouping grouper, i
                 .chooseTasks(inputTaskId, Lists.newArrayList());
 
             // Validate a single task id return
-            assertNotNull("Not null taskId list returned", taskIds);
-            assertEquals("Single task Id returned", 1, taskIds.size());
+            assertNotNull(taskIds, "Not null taskId list returned");
+            assertEquals(1, taskIds.size(), "Single task Id returned");
```

Comment:
error message should negate the assertion

Reply:
done